src/components/AppHeader/PlatformSwitcher/index.tsx

```diff
@@ -9,16 +9,16 @@ export const PlatformSwitcher = () => {
         <UIPlatformSwitcher
             bottomLinkLabel={localize('Looking for CFDs? Go to Trader’s Hub')}
             buttonProps={{
-                className: 'hover:bg-transparent xl:hover:bg-[#e6e9e9] px-[1.6rem]',
+                className: 'hover:bg-transparent lg:hover:bg-[#e6e9e9] px-[1.6rem]',
                 icon: platformsConfig[0].buttonIcon,
             }}
-            itemsWrapperClassName='top-48 h-full xl:top-[4.7rem] xl:h-auto'
-            overlayClassName='top-48 xl:top-[4.7rem]'
+            itemsWrapperClassName='top-48 h-full lg:top-[4.7rem] lg:h-auto'
+            overlayClassName='top-48 lg:top-[4.7rem]'
         >
             {platformsConfig.map(({ active, description, href, icon }) => (
                 <PlatformSwitcherItem
                     active={active}
-                    className='py-[1.4rem] px-[1.6rem] my-[1.4rem] mx-[1.6rem] h-auto xl:py-[2.4rem] xl:px-[1.6rem] xl:m-[1.6rem] xl:mt-[2.4rem] xl:h-[14.3rem]'
+                    className='py-[1.4rem] px-[1.6rem] my-[1.4rem] mx-[1.6rem] h-auto lg:py-[2.4rem] lg:px-[1.6rem] lg:m-[1.6rem] lg:mt-[2.4rem] lg:h-[14.3rem]'
```

Comment:
can we shorten this?
py-[1.4rem] px-[1.6rem] => p-[1.6rem]
same for m as well.
also we have px-[1.6rem] and lg:px-[1.6rem].. does that actually make a difference ? 🤔